bifacial_radiance/bifacial_radiance.py

```diff
@@ -2462,249 +2794,446 @@ def saveResults(self, data, reardata=None, savefile=None):
+                print ("sceneDict warning: 'hub_height' and 'clearance_height"+
+                       "' passed to moduleAnalysis(). Using 'hub_height' "+
+                       "instead of 'clearance_height'")
         else:
-            if clearanceheight is not None:
-                height = clearanceheight + 0.5* np.sin(abs(tilt) * np.pi / 180) * sceney - offset*np.sin(abs(tilt)*np.pi/180) 
+            if 'clearance_height' in sceneDict:
+                height = sceneDict['clearance_height'] + 0.5* \
+                    np.sin(abs(tilt) * np.pi / 180) * \
+                    sceney - offset*np.sin(abs(tilt)*np.pi/180)
+
+                if 'height' in sceneDict:
+                    print("sceneDict warning: 'height' is deprecated, using"+
+                          " 'clearance_height' for moduleAnalysis()")
+                    del sceneDict['height']
             else:
-                print("Pass either hubheight or clearanceheight")
-        '''
-        
-        if modWanted == 0 or rowWanted ==0:
-            print( " FYI Modules and Rows start at index 1."  )
-        
+                if 'height' in sceneDict:
+                    print("sceneDict warning: 'height' is deprecated. "+
+                          "Assuming this was clearance_height that was passed"+
+                          " as 'height' and renaming it in sceneDict for "+
+                          "moduleAnalysis()")
+                    height = sceneDict['height'] + 0.5* np.sin(abs(tilt) * \
+                                      np.pi / 180) * sceney - offset * \
+                                      np.sin(abs(tilt)*np.pi/180)
+                else:
+                    print("Isue with moduleAnalysis routine. No hub_height "+
+                          "or clearance_height passed (or even deprecated "+
+                          "height!)")
+
+        if debug:
+            print("For debug:\n hub_height, Azimuth, Tilt, nMods, nRows, "+
+                  "Pitch, Offset, SceneY, SceneX")
+            print(height, azimuth, tilt, nMods, nRows,
+                  pitch, offset, sceney, scenex)
+
+        if modWanted == 0:
+            print( " FYI Modules and Rows start at index 1. Reindexing to modWanted 1"  )
+            modWanted = modWanted+1  # otherwise it gives results on Space.
+
+        if rowWanted ==0:
+            print( " FYI Modules and Rows start at index 1. Reindexing to rowWanted 1"  )
+            rowWanted = rowWanted+1
+
         if modWanted is None:
             modWanted = round(nMods / 2.0)
         if rowWanted is None:
             rowWanted = round(nRows / 2.0)
-            
-        # Adjusting because modules and rows numbering starts at 0, not 1.
-        
-            
-        #if modwanted > 0:
-        #   modwanted = modwanted - 1
-        
-        #if rowwanted > 0:
-        #    rowwanted = rowwanted - 1
-            
-        
+
         if abs(np.tan(azimuth*dtor) ) <=1 or abs(np.tan(azimuth*dtor) ) > 1:
```

Comment:
What is this line here doing?  isn't tan(azimuth) always either going to be <=1 or >1?

Reply:
I've removed it - must have been left over from the previous revision.

Reply:
I think this is from many, mano moons before.